### PR TITLE
chore/CN-3121: Downgrade the project to .NET Framework 4.7.2

### PR DESCRIPTION
**Pull request by felipefadul**
chore: Downgrade the project to .NET Framework 4.7.2

The main goal is to make possible to use this lib to mask PII data in our integrations that have not yet been migrated to .NET Core.

All tests are passing:

image

Issue: CN-3121

### CODE REVIEW

File: src/JsonDataMaskingFramework/packages.config [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="DeepCloner" version="0.10.4" targetFramework="net47" />
```
Comment (gabrielHosino) [inline src/JsonDataMaskingFramework/packages.config]: Essa dependência aqui acho que tem que atualizar para o 472 também né? Ou não tem problema deixar ela no 47?

File: src/JsonDataMaskingFramework.Test/packages.config [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  <package id="System.Threading.Tasks.Extensions" version="4.5.4" targetFramework="net47" />
+  <package id="System.ValueTuple" version="4.5.0" targetFramework="net47" />
+  <package id="xunit" version="2.4.1" targetFramework="net47" />
+  <package id="xunit.abstractions" version="2.0.3" targetFramework="net47" />
+  <package id="xunit.analyzers" version="0.10.0" targetFramework="net47" />
+  <package id="xunit.assert" version="2.4.1" targetFramework="net47" />
+  <package id="xunit.core" version="2.4.1" targetFramework="net47" />
+  <package id="xunit.extensibility.core" version="2.4.1" targetFramework="net47" />
+  <package id="xunit.extensibility.execution" version="2.4.1" targetFramework="net47" />
+  <package id="xunit.runner.visualstudio" version="2.4.3" targetFramework="net47" developmentDependency="true" />
```
Comment (gabrielHosino) [inline src/JsonDataMaskingFramework.Test/packages.config]: O target para esses pacotes não teria que ser 472 também?

File: src/JsonDataMaskingFramework/packages.config (hunk 1 above)
Comment (felipefadul) [reply src/JsonDataMaskingFramework/packages.config]: Eu havia testado e os testes estavam passando. Mas por via das dúvidas, atualizei aqui [nesse commit](https://github.com/vtex/json-data-masking-framework/pull/1/commits/d0ff04853eb7f0c37e64d9c3ee97a5e52f7f4ca9). Rodei um `Update-Package -Reinstall` no `Package Manager Console` ([fonte](https://stackoverflow.com/questions/36023982/nuget-re-targeting-after-upgrading-from-net-framework-4-5-to-4-6-1)).

File: src/JsonDataMaskingFramework.Test/packages.config (hunk 2 above)
Comment (felipefadul) [reply src/JsonDataMaskingFramework.Test/packages.config]: Eu havia testado e os testes estavam passando. Mas por via das dúvidas, atualizei aqui [nesse commit](https://github.com/vtex/json-data-masking-framework/pull/1/commits/d0ff04853eb7f0c37e64d9c3ee97a5e52f7f4ca9). Rodei um `Update-Package -Reinstall` no `Package Manager Console` ([fonte](https://stackoverflow.com/questions/36023982/nuget-re-targeting-after-upgrading-from-net-framework-4-5-to-4-6-1)).

**Review by gabrielHosino — APPROVED** [review]

**Review by MrCeccatto — APPROVED** [review]

**Pull request merged**